kaiax/gov/headergov/impl/init.go

```diff
@@ -82,6 +82,27 @@ func (h *headerGovModule) Init(opts *InitOpts) error {
 
 func (h *headerGovModule) Start() error {
 	logger.Info("HeaderGovModule started")
+
+	go func() {
+		lastInsertedBlockPtr := ReadLastInsertedBlock(h.ChainKv)
+		if lastInsertedBlockPtr == nil {
+			panic("last inserted block must exist")
+		}
+
+		lastInsertedBlock := *lastInsertedBlockPtr
+		for lastInsertedBlock > 0 {
+			voteBlocks := h.scanAllVotesInHeader(calcEpochIdx(lastInsertedBlock, h.epoch))
+			for blockNum, vote := range voteBlocks {
+				h.cache.AddVote(calcEpochIdx(blockNum, h.epoch), blockNum, vote)
+				InsertVoteDataBlockNum(h.ChainKv, blockNum)
+			}
+
+			WriteLastInsertedBlock(h.ChainKv, lastInsertedBlock)
+			logger.Info("Scanned votes in header", "num", lastInsertedBlock)
+			lastInsertedBlock -= 604800
```

Comment:
Get this number from calcEpochIndex. e.g. `c`alcEpochIdx(lastInsertedBlock, h.epoch)*h.epoch`

Reply:
fixed e004682